Presentation.Web/app/components/it-advice/it-advice-modal-dialog.ts

```diff
@@ -155,12 +155,16 @@
                     $scope.checkErrStart = (startDate, endDate) => {
                         $scope.errMessage = "";
                         $scope.startDateErrMessage = "";
-                        $scope.curDate = new Date();
+                        $scope.curDate = getCurrentDate();
                         if (!moment($scope.startDate, "dd-MM-yyyy").isValid() ||
                             $scope.startDate == undefined) {
                             $scope.startDateErrMessage = "Fra Dato er ugyldig!";
                             return false;
                         }
+                        if (dateString2Date($scope.startDate) < $scope.curDate) {
```

Comment:
Har du set på hvad "moment" modulet er for noget? Det kunne muligvis benyttes til nogle ting her. Se evt. de ændringer jeg har i https://github.com/Strongminds/kitos/pull/474